scalableminds/webknossos
# cleanup erroneous uploads

- additionally, this pr reserves dataset names after the first upload request, so no other upload with the same name can be started

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- upload erroneous dataset
- upload fails
- upload a valid dataset with the same name
- upload should work

### Issues:
- fixes #4971 

------
